python/ray/dataframe/index_metadata.py

```diff
@@ -128,6 +126,37 @@ def _set_index(self, new_index):
 
     index = property(_get_index, _set_index)
 
+    def _get_index_cache(self):
+        """Get the cached Index object, which may sometimes be an OID.
+
+        This will ray.get the Index object out of the Ray store lazily, such
+        that it is not grabbed until it is needed in the driver. This layer of
+        abstraction is important for allowing this object to be instantiated
+        with a remote Index object.
+
+        Returns
```

Comment:
Can we add a `:` after `Returns`?

Probably don't need to wait for Travis after that.